g11n-ws/modules/md-restful-api-i18n/src/main/java/com/vmware/vip/i18n/api/base/TranslationProductAction.java

```diff
@@ -103,13 +104,7 @@ private TranslationDTO getAllCompTrans(String productName, String version,String
     
     public APIResponseDTO getMultTrans(String productName, String version, String componentsStr, String localesStr, String pseudo,
             HttpServletRequest req)  throws Exception {
-        if(StringUtils.isEmpty(componentsStr) && StringUtils.isEmpty(localesStr)) {
-            return  super.handleResponse(APIResponseStatus.OK, getAllCompTrans( productName,  version, pseudo, req)) ;
-        }else {
             return getPartialComTrans(productName, componentsStr,  version,  localesStr,  pseudo, req);
-        }
-        
-        
```

Comment:
Why did you remove these lines? What if StringUtils.isEmpty(componentsStr) && StringUtils.isEmpty(localesStr) ?

Reply:
As Colin's recommend change the design. the new design have this feature